### DISCUSSION TITLE
Display SearchBar with TabView

### DISCUSSION THREAD

**Discussion by lam0620** [react-native-community/react-native-tab-view#1215] [Q&A]
Does anyone know how to display Searchbar (react-native-elements) with TabView as below mock?

![image](https://user-images.githubusercontent.com/12405565/80181190-92ef5f00-862e-11ea-8822-d3cf565310b2.png)

I can display the TabView by
```
      <TabView
        navigationState={{ index, routes }}
        renderScene={renderScene}
        onIndexChange={setIndex}
        renderTabBar={renderTabBar}
        initialLayout={initialLayout}
      />
```
But if I do
```
<View>
      <SearchBar
        placeholder="Type Here..."
        onChangeText={this.updateSearch}
        value={search}
      />
      <TabView
        navigationState={{ index, routes }}
        renderScene={renderScene}
        onIndexChange={setIndex}
        renderTabBar={renderTabBar}
        initialLayout={initialLayout}
      />
</View>
```
It just show SearchBar only.
I'm newbie react native. it'd appreciated if anyone help me.

Thank you!

**Comment by jagannathBhat** [comment]
I tried implementing this with TextInput instead of SearchBar and it worked. Go to the link below for the code.
[https://snack.expo.io/@jagannathbhat/react-native-tab-view-with-searchbar](https://snack.expo.io/@jagannathbhat/react-native-tab-view-with-searchbar)
